src/components/Personsøk/Personsøk.tsx

```diff
@@ -20,7 +23,7 @@ import styles from './personsøk.module.less';
 
 interface Props {
     person: ApiResult<Person>;
-    onFetchByFnr: (fnr: string) => void;
+    onFetchByFnr?: (fnr: string) => void; //TODO: burde fjernes
```

Comment:
Dele opp personsøk komponenten i to? Denne blir sendt med fordi en random annen komponent skal ha søkemuligheten også men får med all den andre logikken på kjøpet  :/